indico/modules/rb/client/js/common/bookings/BookingEditForm.jsx

```diff
@@ -384,6 +389,12 @@ class BookingEditForm extends React.Component {
             disabled={submitSucceeded}
           />
         </Segment>
+        {renderPluginComponents('rb-booking-form-extra-fields', {
+          room,
+          booking,
+          formProps,
```

Comment:
I'm replacing this (which you do not use anyway in your plugin) with `disabled` (`submitting || submitSucceeded`) here and adding this in booking creation. You should use that in your plugin to disable the custom fields.

Reply:
ok, will do! 👍